ci/ci/constants.py

```diff
@@ -28,5 +28,6 @@ def __init__(self, gh_username: str, hail_username: Optional[str] = None, teams:
     User('ehigham', 'ehigham', HAIL_TEAM),
     User('jkgoodrich', 'jgoodric'),
     User('konradjk', 'konradk'),
+    User('ksittig', 'ksittig', HAIL_TEAM),
```

Comment:
First item is github username and `kasittig` is not `ksittig`

Reply:
Incidentally, this will change the sort order again.

Reply:
Are you signing yourself up for reviews intentionally? If not, don't pass a third argument to `User`?

Reply:
> Are you signing yourself up for reviews intentionally? If not, don't pass a third argument to `User`.

I am! It seems like we'd really benefit from having more reviewers and I'm happy to help.